sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample7_AnalyzeHealthcareEntitiesBatchAsync.cs

```diff
@@ -0,0 +1,120 @@
+                                minimal ST depressions in the anterior lateral leads , thought due to fatigue and wrist pain , his anginal equivalent. Due to the patient's \
+                                increased symptoms and family history and history left main disease with total occasional of his RCA was referred for revascularization with open heart surgery.";
+
+            string document2 = "Prescribed 100mg ibuprofen, taken twice daily.";
+
+            List<TextDocumentInput> batchInput = new List<TextDocumentInput>()
+            {
+                new TextDocumentInput("1", document1)
+                { Language = "en" },
+                new TextDocumentInput("2", document2)
+                { Language = "en" },
+                new TextDocumentInput("3", string.Empty)
+            };
+
+            AnalyzeHealthcareEntitiesOptions options = new AnalyzeHealthcareEntitiesOptions()
+            {
+                IncludeStatistics = true
+            };
+
+            AnalyzeHealthcareEntitiesOperation healthOperation = await client.StartAnalyzeHealthcareEntitiesAsync(batchInput, options);
+
+            await healthOperation.WaitForCompletionAsync();
+
+            Console.WriteLine($"AnalyzeHealthcareEntities operation was completed");
+
+            Console.WriteLine($"Created On   : {healthOperation.CreatedOn}");
+            Console.WriteLine($"Expires On   : {healthOperation.ExpiresOn}");
+            Console.WriteLine($"Id           : {healthOperation.Id}");
+            Console.WriteLine($"Status       : {healthOperation.Status}");
+            Console.WriteLine($"Last Modified: {healthOperation.LastModified}");
+
+            foreach (AnalyzeHealthcareEntitiesResultCollection documentsInPage in healthOperation.GetValues())
+            {
+                Console.WriteLine($"Results of Azure Text Analytics \"Healthcare\" Model, version: \"{documentsInPage.ModelVersion}\"");
+                Console.WriteLine("");
+
+                int i = 0;
+
+                foreach (AnalyzeHealthcareEntitiesResult result in documentsInPage)
+                {
+                    TextDocumentInput document = batchInput[i++];
+
+                    Console.WriteLine($"On document (Id={document.Id}, Language=\"{document.Language}\"):");
+
+                    if (result.HasError)
+                    {
+                        Console.WriteLine("  Error!");
+                        Console.WriteLine($"  Document error code: {result.Error.ErrorCode}.");
+                        Console.WriteLine($"  Message: {result.Error.Message}");
+                    }
+                    else
+                    {
+                        Console.WriteLine($"    Recognized the following {result.Entities.Count} healthcare entities:");
+
+                        foreach (HealthcareEntity entity in result.Entities)
```

Comment:
we should add new properties to the sample like normalized text, assertions, and relations.

This applies to other sample files